source/layouts/blog.erb

```diff
@@ -18,23 +18,121 @@
+    <header class="site-header">
       <div class="site-header__brand">
         <a class="site-header__link" href="https://www.unboxedconsulting.com/">
-          Unboxed
+        Unboxed
         </a>
       </div>
-    </div>
+    </header>
+
+    <div class="blog-page">
+      <header class="blog-header__container">
+        <div class="blog-header__article-header">
+          <h1 class="blog-header__article-title">
+            <%= current_page.data.title %>
+          </h1>
+          <a class="blog-header__article-author" href="">
+            <%= current_page.data.author %>
+          </a>
+          <p class="blog-header__article-published-date">
+            <span>
+            <%= Time.parse(current_page.data.date).strftime("%B %e, %Y") %>
+          </span>
+          </p>
+          <div class="blog-header__article-tags blog-header__article-tag-border">
+          <% data.tags.each do |tag| %>
+            <div class="blog-header__article-tag-link
+              <%= "blog-header__article-tag-active blog-header__article-tag-border-active" if current_article.tags.include?(tag) %>">
+                <a href=""><%= tag %></a>
+              </div>
+            <% end %>
+          </div>
+        </div>
+      </header>
+
+      <section class="blog-article">
+        <article class="blog-article__container">
+          <div class="blog-article__content">
+            <%= yield %>
+          </div>
+        </article>
+      </section>
+
+      <section class="blog-social-icons">
+        <div class="blog-social-icons__container">
+          <a class="social-icons__icon-container" href="http://twitter.com/Ubxd" target="_blank">
+            <div class="social-icons__icon">
+              <%= image_tag 'twitter.png', class: 'social-icons__image', alt: 'Twitter' %>
+            </div>
+          </a>
+          <a class="social-icons__icon-container" href="https://www.facebook.com/pages/Unboxed-Consulting/114802528557320" target="_blank">
+            <div class="social-icons__icon">
+              <%= image_tag 'facebook.png', class: 'social-icons__image', alt: 'Facebook' %>
+            </div>
+          </a>
+          <a class="social-icons__icon-container" href="http://www.linkedin.com/company/unboxed-consulting" target="_blank">
+            <div class="social-icons__icon">
+              <%= image_tag 'linkedin.png', class: 'social-icons__image', alt: 'LinkedIn' %>
+            </div>
+          </a>
+        </div>
+      </section>
+
+      <section class="blog-author">
+        <div class="blog-author__container">
+          <div class="blog-author__image">
+            <img src="<%= data.people.graeme.image %>" alt="blog author"/>
+          </div>
+          <aside class="blog-author__right-section">
+            <p class="blog-author__name">
+              <span><%= data.people.graeme.name %></span>
+            </p>
+            <div class="blog-author__bio">
+              <%= data.people.graeme.bio %>
```

Comment:
Could we make this dynamic? It won't always be Graeme - perhaps a lookup on the data file based on the author's name